Makefile

```diff
@@ -1,13 +1,20 @@
-.PHONY: build-all clean swagger
+.PHONY: build-all clean swagger release
 BIN = $(addprefix bin/,$(shell ls cmd))
-COMMIT := $(shell git rev-parse --short HEAD)
+COMMIT ?= $(shell git rev-parse --short HEAD)
 DATE := $(shell date)
 
 build-all: $(BIN)
 
 bin/%: cmd/%/main.go
 	GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -ldflags="-X 'github.com/sapcc/archer/internal/config.Version=$(COMMIT)' -X 'github.com/sapcc/archer/internal/config.BuildTime=$(DATE)'" -o $@ $<
 
+release:
+	GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'github.com/sapcc/archer/internal/config.Version=$(COMMIT)' -X 'github.com/sapcc/archer/internal/config.BuildTime=$(DATE)'" -o bin/archerctl_darwin_adm64 cmd/archerctl/main.go
+	GOOS=darwin GOARCH=arm64 go build -ldflags="-X 'github.com/sapcc/archer/internal/config.Version=$(COMMIT)' -X 'github.com/sapcc/archer/internal/config.BuildTime=$(DATE)'" -o bin/archerctl_darwin_arm64 cmd/archerctl/main.go
+	GOOS=windows GOARCH=amd64 go build -ldflags="-X 'github.com/sapcc/archer/internal/config.Version=$(COMMIT)' -X 'github.com/sapcc/archer/internal/config.BuildTime=$(DATE)'" -o bin/archerctl.exe cmd/archerctl/main.go
+	GOOS=linux GOARCH=amd64 go build -ldflags="-X 'github.com/sapcc/archer/internal/config.Version=$(COMMIT)' -X 'github.com/sapcc/archer/internal/config.BuildTime=$(DATE)'" -o bin/archerctl_linux_x86_64 cmd/archerctl/main.go
```

Comment:
Those should ideally include -s -w

Reply:
`-s` strips all debug symbols and makes it unfeasible to debug the binary in delve/gdb
`-w` surpresses all warning, why should anyone want this?

Reply:
I meant big -W

It makes binaries significantly smaller and in prod you are normally not attaching a debugger anyway.

Reply:
> I meant big -W
```
[qa-de-1:monsoon3][~]$ ld -W
ld: unknown option: -W
```
🤔 
> 
> It makes binaries significantly smaller and in prod you are normally not attaching a debugger anyway.

I did because a binary crashed and was very upset that I've had to fetch and rebuild the binary to debug it for 2 min.
Will minimize it as soon as someone complains that it won't fit on their floppy drive. :)

Reply:
Have to correct myself, it seems go has it's own linker (thought always it uses gnu/bsd ld)

```
# go tool link -h
  -s	disable symbol table
...
  -w	disable DWARF generation
```